### PR TITLE
Add  dutch translation

### PR DESCRIPTION
**Pull request by Metal-Eagle**
Added dutch translation

### CODE REVIEW

File: src/localize.ts [hunk 1]
```diff
@@ -19,6 +21,7 @@ const languages: Record<string, unknown> = {
     "pt-BR": pt_BR,
     sv,
     "zh-Hans": zh_Hans,
+    "nl": nl,
```
Comment (piitaya) [inline src/localize.ts:24]:
```suggestion
    nl,
```
No need to repeat the `nl` key.

**Review by piitaya — CHANGES_REQUESTED** [review]
Can you fix the conflict with the main branch ? It's good for me, I will merge when it's done 🙂

**Comment by thibmaek** [comment]
As a native Dutch speaker lots of these seem incorrect to me. 
For example:
- laatste gewijzigd should be laatst gewijzigd
- horizontaal layout should be horizontale layout
- justify is uitlijnen in dutch

File: src/localize.ts (hunk 1 above)
Comment (Metal-Eagle) [reply src/localize.ts]: I edited this, it should look better now.

**Review by piitaya — APPROVED** [review]

**Pull request merged**